app/femr/ui/models/manager/IndexViewModelGet.java

```diff
@@ -33,5 +45,108 @@ public String getToday()
+        this.encounterItem = patientEncounter;
+    }
+    // gets array list item of Patient Encounter Items
 
+    public PatientEncounterItem getEncounter (int i){
+        return encounterItem.get(i);
+    }
+
+    // gets array list of Patient Encounter Items
+    public List<PatientEncounterItem> getEncounter() {
+        return encounterItem;
+    }
+
+//returns turn around time in the form of a String
+    public String turnAroundTime(PatientEncounterItem item){
+        PatientEncounter patientEncounter = new PatientEncounter();
+        String time=" ";
+        if(item.getPharmacyDateOfVisit()!= null &&item.getTriageDateOfVisit()!=null) {
+            SimpleDateFormat ft = new SimpleDateFormat("HH:mm:ss");
+            try {
+                Date td,pd;
+                td = ft.parse(getTriageTime(item));
+                pd= ft.parse(getPharmTime(item));
+                long diff = pd.getTime() - td.getTime();
+                String ss = String.valueOf(diff / 1000 % 60);
+                String sm = String.valueOf(diff / (60 * 1000) % 60);
+                String sh = String.valueOf(diff / (60 * 60 * 1000));
+                time="Hours: "+sh+" Minutes: " +sm+"  Seconds: "+ss;
+            }catch (ParseException e) {
+                System.out.println("Unparseable using " + ft);
+            }
+            return time;
+        }
+        else {
+            return time;
+        }
+    }
+
+    public void setHPI(List<Map<String, TabFieldItem>> tab) {
+        this.tab = tab;
+    }
+    // gets array list item of HPI Tab Mapped Items
+
+    public Map<String, TabFieldItem>  getHPI(int i){
+        return tab.get(i);
+    }
+
+    // gets array list of HPI Items
+    public List<Map<String, TabFieldItem>> getHpi() {
+        return tab;
+    }
+
+    public void setVitals(List<VitalMultiMap> vital) {
+        this.vitalItems = vital;
+    }
+    // gets array list item of vital Items
+
+    public VitalMultiMap getVitals (int i){
+        return vitalItems.get(i);
+    }
+
+    // gets array list of Vital Items
+    public List<VitalMultiMap> getVital() {
+        return vitalItems;
+    }
+    PatientEncounter patientEncounter = new PatientEncounter();
+    public String getPharmTime(PatientEncounterItem item) {
+        String pharmTime;
+        if(item.getPharmacyDateOfVisit()!=null) {
+            String[] parts = item.getPharmacyDateOfVisit().split("-");
+            pharmTime = parts[1];
+            return pharmTime;
+        }
+        else
+            return " ";
+    }
+
+    public String getTriageTime(PatientEncounterItem item) {
+        String triageTime;
+        if(item.getTriageDateOfVisit()!=null) {
+            String[] parts = item.getTriageDateOfVisit().split("-");
+            triageTime = parts[1];
+            return triageTime;
+        }
+        else
+            return " ";
+    }
+    public String getMedicalTime(PatientEncounterItem item) {
```

Comment:
looking a bit deeper, only patients with more than one chief complaint are being displayed because of the conditional if statement that this is nested in:
https://github.com/FEMR/femr/pull/563/files#diff-559b23d6ab77babeb5d791a98ed6d4c6R96

It is hard to  remember what "p", "v", and "i" all stand for when reading the code as someone who didn't write it :) . while technically correct, it's probably worth making the variable names meaningful